src/api/controllers/health-controller.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * PURPOSE: Health Endpoint Controller for Project Wayfinder
+ */
+import { Get, Route, SuccessResponse, Tags, Example } from "tsoa";
+
+  interface HealthPointInterface {
+    response: string
+  }
+  
+  @Route("health")
+  @Tags("Health")
+  export class HealthClassController {
```

Comment:
Does this need to `extends Controller` ? Thats just what I see in the documentation for tsoa

Reply:
Fixed to avoid potential future issues